src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -526,5 +487,25 @@ def _insertContainerRulesT0(self):
         # Register the result in DBSBuffer
         if subscriptionsMade:
             self.markSubscribed.execute(subscriptionsMade)
+            logging.info("%d containers successfully locked in Rucio and local database", len(subscriptionsMade))
 
         return
+
+    def _activityMap(self, rseName):
+        """
+        It maps the WMAgent type (Production vs T0) and the RSE name to
+        properly set the rule activity field
+        :param rseName: a string with the RSE name
+        :return: a string with the rule activity
+        """
+        if not self.isT0agent and not rseName.endswith("_Tape"):
+            return "Production Output"
+        elif self.isT0agent and rseName.endswith("_Tape"):
+            return "T0 Tape"
+        elif self.isT0agent:
+            return "T0 Export"
+        else:
+            msg = "This code should never be reached. Report it to the developers. "
+            msg += "Trying to create container rule for WMAgent: {}, with RSE name: {}".format(self.isT0agent,
```

Comment:
Is the first argument here right? At this point we know `self.isT0agent` is false.

Reply:
@germanfgv did you mean L505? If not, please point me to the exact line or statement you're referring to.

Reply:
I'm referring to L509. The Exception message.

Reply:
You're right. Let me update this code.